PayItForward.Web/Models/AdminViewModels/IndexViewModel.cs

```diff
@@ -1,20 +1,23 @@
-﻿namespace PayItForward.Web.Models.HomeViewModel
+﻿namespace PayItForward.Web.Models.AdminViewModels
 {
+    using System;
     using System.Collections.Generic;
-    using PayItForward.Web.Models.StoriesViewModel;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using PayItForward.Web.Models.StoryViewModels;
 
     public class IndexViewModel
     {
         public IEnumerable<BasicStoryViewModel> Stories { get; set; }
 
-        public IEnumerable<BasicStoryViewModel> Users { get; set; }
-
         public int CurrentPage { get; set; }
 
         public int TotalPages { get; set; }
 
         public string CurrentUrl { get; set; }
 
         public string SearchWord { get; set; }
+
+        public Guid Id { get; set; }
```

Comment:
Change the id name with for which object the id is, so it can be more readable, because the name of the view model is index and it is not clear what is this id for

Reply:
As I was watching this model class.... it's not used, so I will delete it